src/realtime_clock.cpp

```diff
@@ -49,10 +48,9 @@ namespace realtime_tools
      system_time_(ros::Time()), 
      last_realtime_time_(ros::Time()),
      running_(true),
-     initialized_(false)
+     initialized_(false),
```

Comment:
Do you want to keep this style of initialisation or should we adopt the newer trend of initialising them in the class definition? It'd limit the scope of this constructor to the thread setup.

Reply:
Moved initial values to class definition in 30109311ac8c8a88887f56f2ac20c08d1f47f0b0